corehq/ex-submodules/pillowtop/reindexer/change_providers/case.py

```diff
@@ -0,0 +1,49 @@
+import itertools
+
+from casexml.apps.case.models import CommCareCase
+from corehq.form_processor.backends.sql.dbaccessors import CaseAccessorSQL
+from corehq.form_processor.change_providers import _sql_case_to_change
+from corehq.form_processor.utils.general import should_use_sql_backend
+from pillowtop.reindexer.change_providers.couch import CouchViewChangeProvider
+from pillowtop.reindexer.change_providers.interface import ChangeProvider
+
+
+class CouchDomainCaseChangeProvider(CouchViewChangeProvider):
+    def __init__(self, domain):
```

Comment:
is there any reason for this to be a class instead of a function? e.g. `get_couch_case_change_provider(domain)`